lib/ansible/modules/cloud/docker/docker_node_facts.py

```diff
@@ -70,12 +96,14 @@
     description:
       - Facts representing the current state of the node. Matches the C(docker node inspect) output.
       - Will be C(None) if node does not exist.
+      - Will contain multiple entries if more than one node provided in I(name).
+      - If I(name) contains list of nodes, the output will contain information only about registered nodes.
     returned: always
     type: dict
```

Comment:
That's no longer true. You return a `list`. Also, the name is misleading, since these are not facts about one node, but a list of facts of nodes. Maybe use `nodes_facts`, or simply `nodes`?

Also, `exists` doesn't make sense any more when this is a list.